ui/dashboard.js

```diff
@@ -78,10 +78,32 @@ function create_box (id, icon, number, currency_icon) {
     settings.page_index = $('#page-wrapper').html();
 }
 
+function update_eth_node_conn_status_ui(is_local_eth) {
```

Comment:
please use full names not abbreviation. So here: `update_eth_node_connection_status_ui`